binder/environment.yml

```diff
@@ -38,7 +38,7 @@ dependencies:
 - qtawesome >=0.5.7
 - qtconsole >=5.0.1
 - qtpy >=1.5.0
-- rtree >=0.8.3
+- rtree >=0.9.6
```

Comment:
Why `0.9.6` and not `0.9.7`? I say it because those versions were released only a few days apart:

https://pypi.org/project/Rtree/#history

So probably `0.9.7` includes fixes that the Rtree maintainers missed in `0.9.6`

Reply:
Correct. This should be changed to 0.9.7.